production/operator/crds/monitoring.grafana.com_grafanaagents.yaml

```
@@ -1054,6 +1054,16 @@ spec:
                 required:
                 - host
                 type: object
+              args:
+                default:
+                - -config.file=/var/lib/grafana-agent/config/agent.yml
+                - -config.expand-env=true
+                - -reload-port=8081
+                description: Args controls command line additional arguments sent
+                  to the Agent pod.
+                items:
+                  type: string
+                type: array
               configMaps:
                 description: ConfigMaps is a liset of config maps in the same namespace
```

```suggestion
                description: ConfigMaps is a list of config maps in the same namespace
```

This is unrelated to your change but I just noticed this typo